examples/using-mobx/src/components/layout.js

```diff
@@ -1,32 +1,32 @@
-import React, { Component } from "react"
+import React from "react"
+import PropTypes from "prop-types"
 import { Link } from "gatsby"
 import Counter from "./Counter"
 
-import DevTools from "mobx-react-devtools"
-
-class DefaultLayout extends Component {
-  render() {
-    return (
-      <div>
-        <Link to="/">
-          <h3>MobX example</h3>
-        </Link>
-        <DevTools />
-        <Counter />
-        <ul>
-          <li>
-            <Link to="/a/">a</Link>
-          </li>
-          <li>
-            <Link to="/b/">b</Link>
-          </li>
-          <li>
-            <Link to="/c/">c</Link>
-          </li>
-        </ul>
-        {this.props.children}
-      </div>
-    )
-  }
+const DefaultLayout = props => {
```

Comment:
Why not go for destructring here?

```suggestion
const DefaultLayout = ({ children, ...props })=> {
```

Reply:
didn't remember that. it can be destructured, i'll change it and push the update in a bit